src/Go/effects/netscriptGoImplementation.ts

```diff
@@ -322,6 +332,49 @@ export function getStats() {
   return statDetails;
 }
 
+export const boardValidity = {
```

Comment:
I don't think you're using this export anywhere? In which case, you can also change this to a tighter
```js
const boardValidity = {
...
} const;
```
Declaration (making it a true constant for TypeScript)